edgemesh/pkg/server/ip.go

```diff
@@ -0,0 +1,26 @@
+package server
+
+import (
+	"net"
+	"time"
+
+	"github.com/kubeedge/beehive/pkg/common/log"
+)
+
+// getIP returns the specific interface ip of version 4
+func getIP() (net.IP, error) {
+	for {
+		ifaces, err := net.InterfaceByName(inter)
+		if err != nil {
+			return nil, err
+		}
+		addrs, _ := ifaces.Addrs()
+		for _, addr := range addrs {
+			if ip, inet, _ := net.ParseCIDR(addr.String()); len(inet.Mask) == 4 {
+				return ip, nil
+			}
+		}
+		log.LOGGER.Warnf("the interface" + inter + "have not config ip of version 4")
```

Comment:
same here

Reply:
please use `%s` instead of `+ inter +` the definition of the verbs can be found here: https://godoc.org/fmt